index.html

```diff
@@ -37,154 +36,250 @@ <h1> Hey there. I’m Alex.<span class="green" > <br> I’m a software developer
+    
+                        </li>
+                        <li>
+                            <label for="msg">
+                                <textarea name="user_message" id="msg" cols="30" rows="5" maxlength="500" placeholder="Enter text here" required></textarea>
+                            </label>
+                        </li>
+                        <li class="button back-image">
+                            <button type="submit" class="btn">Get in touch</button>
+                        </li>
+    
+                    </ul>
+    
+                </form>
+    
+                    </div>
+                </section>
 
+
+            </section>
+
+<<<<<<< Updated upstream
            </article>
            </div>
        
+       </section>
+       <section class="form-section">
+        <div class="form-description">
+            <p class="touch">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</p>
+            </div>
+            <div class="contact-form">
+                <form action="https://formspree.io/f/mjvzwlqy" method="post">
+                   <ul>
+                       <li >
+                           <label for="name">
+                               <input type="text" id="name"  name="user_name" placeholder="Full name" maxlength="30" required>
+                           </label>
+                       </li>
+                       <li>
+                           <label for="mail">
+                               <input type="email" id="mail" name="user_email" placeholder="Email address" required>
+                           </label>
+                       </li>
+                       <li>
+                           <label for="msg">
+                               <textarea name="user_message" id="msg" cols="30" rows="5" maxlength="500" placeholder="Enter text here" required></textarea>
+                           </label>
+                       </li>
+                       <li class="button back-image">
+                           <button type="submit" class="btn">Get in touch</button>
+                       </li>
+                   </ul>
+               </form>
+                   </div>
+
        </section>
        </section>
+=======
+            <footer > 
+            <hr class="bottom-line">
+            <ul class ="socials">
+                <li><a href="https://twitter.com/BadruNdawula"><img src="./images/twitter.jpg" alt=""></a></li>
+                <li><a href="https://www.linkedin.com/in/badru-ndawula-32b371236/"><img src="./images/Linkedin icon.jpg" alt=""></a></li>
+                <li><a href="https://github.com/PrinceBadru"><img src="./images/github.jpg" alt=""></a></li>
+                <li><a href="#"><img src="./images/medium.jpg" alt=""><a></li>
+                <li><a href="#"><img src="./images/angellist.jpg" alt=""></a></li>
+            </ul> 
+            </footer>
+            
+>>>>>>> Stashed changes
```

Comment:
- [ ] Kindly remove this comments for cleaner codebase. 👍 Please do the same for line 234 and  and 270 as well. 😄